paper-server/src/main/java/io/papermc/paper/command/brigadier/ApiMirrorRootNode.java

```diff
@@ -124,7 +124,12 @@ public static void validatePrimitiveType(ArgumentType<?> type) {
                 }
 
                 converted = this.unwrapArgumentWrapper(pureArgumentNode, customArgumentType, customArgumentType.getNativeType(), suggestionProvider);
-            } else if (pureArgumentType instanceof final VanillaArgumentProviderImpl.NativeWrapperArgumentType<?, ?> nativeWrapperArgumentType) {
+            }
+            // This is a special case, as we override the suggestions here
+            else if (pureArgumentType instanceof final VanillaArgumentProviderImpl.ScoreHolderWrapperArgumentType scoreHolderWrapperArgumentType) {
```

Comment:
please fix the formatting here, this looks terrible :sweat_smile: 
For all I care just move the comment into the elif block.